source/_patterns/03-organisms/card-grid/_card-grid.twig

```diff
@@ -0,0 +1,26 @@
+{#
+  Card Grid!
+
+  This is provided to show you an example of our Cards in grid in Pattern Lab.
+  Additional implementations are app-specific and should be handled on a case-by-case basis.
+
+  see https://getbootstrap.com/docs/4.0/components/card/ for more details
+#}
+
+<div class="row">
+  {% for item_card in card_grid_rows %}
+    {% set item_card = {
+      card_cols: 'col-sm-12 col-md-6 col-lg-4',
```

Comment:
So, I know that cards have this concept of being their own columns in bootstrap 4, but why aren't we doing this with wrapper items instead? Are we gaining something by junking up the card with these extra classes?

Reply:
I think I originally had a wrapper class on the `views-view-unformatted--frontpage.html.twig` template that applied these styles, but we'd chatted about having that template be a bit simpler. FWIW I think this is simpler / easier to read, but I certainly cede to your experience.